tests/generic_views/test_base.py

```diff
@@ -136,9 +136,8 @@ def test_invalid_keyword_argument(self):
         """
         # Check each of the allowed method names
         for method in SimpleView.http_method_names:
-            kwargs = dict(((method, "value"),))
             with self.assertRaises(TypeError):
-                SimpleView.as_view(**kwargs)
+                SimpleView.as_view(method='value')
```

Comment:
Sorry, maybe I'm missing something. How are these not equivalent other than style? Both forms are supplying a single keyword argument named `method` with the value `'value'`, right?